site/reference/schema/kptfile/kptfile.json

```diff
@@ -1,105 +1,125 @@
 {
-  "$schema": "http://json-schema.org/draft/2019-09/schema#",
-  "type": "object",
-  "anyOf": ["#/definitions/Kptfile"],
```

Comment:
i believe we will still need all 3 of these fields in order for language server support to work without making manual modifications to the schema

Reply:
@droot We can add this as part of make generate step, add these static fields automatically after generating schema.

Reply:
this can be done using jq, something similar to the following:

<img width="1290" alt="Screen Shot 2021-09-28 at 2 55 54 PM" src="https://user-images.githubusercontent.com/8933338/135151716-b38200ed-2068-4f36-9328-a21cbea54a64.png">